jekyll/_api/v1-reference.md

```diff
@@ -5,7 +5,7 @@ categories: [reference]
 description: Using the CircleCI API
 ---
 
-The CircleCI API is a RESTful, fully-featured API that allows you to access all information and trigger all actions in CircleCI. **Note:** The following limitations exist:
+The CircleCI API is a JSON, fully-featured API that allows you to access all information and trigger all actions in CircleCI. **Note:** The following limitations exist:
```

Comment:
Would _maybe_ rewrite this to say: "The CircleCI API is a fully-featured JSON API that allows you to access..."

Just to shrink the distance between the _thing_ and the _type of_ thing.